nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -383,22 +403,46 @@ private ValidationResult validateDynamicKCLConfigProperty(final String subject,
                     .valid(false).build();
         }
 
-        final KinesisClientLibConfiguration kclTemp = new KinesisClientLibConfiguration("validate", "validate", null, "validate");
+        final ConfigsBuilder configsBuilderTemp = new ConfigsBuilder(
+                getStreamName(context),
+                getApplicationName(context),
+                KinesisAsyncClient.builder().region(Region.US_WEST_2).build(),
```

Comment:
Should we use `Region.of(context.getProperty(REGION).getValue())` if the property is set (presumably should be)?

At least add a comment here explaining that these builders aren't actually going ot be used beyond validation (to avoid future maintenance confusion)